packages/ui/src/molecules/AggregateRating/AggregateRating.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { forwardRef, Fragment } from 'react'
+import type { FC, HTMLAttributes, ReactNode } from 'react'
+
+import List from '../../atoms/List'
+
+export interface AggregateRatingProps extends HTMLAttributes<HTMLUListElement> {
```

Comment:
Does this suggestion make sense?!

```suggestion
export interface AggregateRatingProps extends ListProps {
```